cmd/cli/metrics.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"io"
+
+	"github.com/spf13/cobra"
+)
+
+const metricsDescription = `
+This command consists of multiple subcommands related to managing metrics
+associated with osm.
```

Comment:
Should we indicate that its prometheus format metrics only as of today

Reply:
The CLI is configuring prometheus scraping, not necessarily the format.
It would make more sense to note that it enables prometheus scraping rather than talking about formats.